airflow/api_fastapi/auth/managers/models/resource_details.py

```diff
@@ -104,3 +104,4 @@ class DagAccessEntity(Enum):
     TASK_LOGS = "TASK_LOGS"
     WARNING = "WARNING"
     XCOM = "XCOM"
+    DAG_VERSION = "DAG_VERSION"
```

Comment:
Should we change `airflow/security/permissions.py` and `fab_auth_manager` as well?

Reply:
Yep I think we are missing a bunch of pieces here. You can take a look at the AssetAlias PR to check how to add an entity.